learning-another-language/hello-world.md

```diff
@@ -0,0 +1,279 @@
+| --------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `console`                                           | A variable. The word "console" usually refers to the Terminal, command line, or wherever output logs get printed. |
+| `console.log( ... )`                                | A function call that takes in some arguments.                                                                    |
+| `'Hello, World!'`                                   | A string literal. This string is surrounded by single-quotes (`'`).                                               |
+| `;`                                                 | A semicolon.                                                                                                      |
+
+Let's debrief in slightly more detail.
+
+### The Console
+
+`console` seems a little random here. It's lowercase, like Python variables, but there seems to be no variable assignment. We don't see any `import`s or anything like that, so where did `console` come from?
+
+`console` is a variable that references a specific object: the console! This console object gives us access to wherever the program prints out output.
+
+It seems that `console` is within scope of our JavaScript file by default.
+
+At this moment, at the beginning of our JavaScript journey, we won't go deeper into this.
+
+### Calling a Function
+
+`console.log()` is calling a function. We might not know all the details about this function, but we know that it's responsible for printing objects to the console.
+
+We may make guesses that:
+
+1. The function being called here is named `log`
+1. The function `log` is a member of `console` (hence the dot-operator)
+1. We use the dot-operator to access members of objects
+1. The parens `()` is how we call and invoke the function
+
+### Single Quotes for Strings
+
+The string literal `'Hello, World!'` is surrounded by single-quotes, instead of double-quotes.
+
+If we experiment and replace it with double-quotes (`console.log("Hello, World!");`), we'll see that our code still runs as expected!
+
+In JavaScript, we can create string literals with either syntax.
+
+### !callout-info
+
+## This Curriculum Will Prefer Single-Quotes
+
+The text in this curriculum will prefer single-quotes. In the grand scheme of things, the most important thing is consistency within a team.
+
+### !end-callout
+
+### Ends With a Semicolon
+
+This line of code ends with a semicolon (`;`).
+
+In JavaScript, semicolons are used to conclude statements.
+
+### !callout-info
+
+## Statements
+
+In programming, a statement is a unit of code that contains an action to execute. We use statements mainly to _perform an action with side effects_. The following are examples of statements:
```

Comment:
Elsewhere in the curriculum, we stress avoiding side effects, so here's a rephrase to avoid possible confusion around the perception of mixed messaging.
```suggestion
In programming, a statement is a unit of code that contains an action to execute. We use statements mainly in the forms of expressions that update values, or control statements that affect the flow of the program. The following are examples of statements:
```